### PR TITLE
WIP - Create edinburgh-uk.md

### LINKED ISSUE #5: EmilyUna65 favorite places to eat
Let's add a document using:
- [x] Create a branch
- [x] Add a file
- [x] Commit my file
- [x] Open Pull Request
- [ ] Discuss
- [ ] Merge

@carolynshin do you have any specific reccommendations?

### PR DESCRIPTION
**Pull request by EmilyUna65**
Resolves #5 by adding a file for places to eat in Edinburgh
closes 
or fixes

### CODE REVIEW

File: eats/edinburgh-uk.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Edinburgh, Scotland Great Places to Eat
+* Fenwicks
+* The Witchery
```
Comment (EmilyUna65) [inline eats/edinburgh-uk.md:3]: Think it's The Witchery by the Castle :smile:

**Comment by AMZod** [comment]
Brilliant :)

File: eats/edinburgh-uk.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+* The Dome
+* Bar One
```
Comment (cowperthwait) [inline eats/edinburgh-uk.md]: @EmilyUna65 is this the place?: [All Bar One](http://www.allbarone.co.uk/national-search/scotland-and-northern-ireland/all-bar-one-george-street-edinburgh)

**Comment by carolynshin** [comment]
http://www.timeout.com/edinburgh/restaurants/the-best-restaurants-in-edinburgh

File: eats/edinburgh-uk.md (hunk 2 above)
Comment (monicanpm) [inline eats/edinburgh-uk.md]: Is there alcohol here?

Comment (EmilyUna65) [inline eats/edinburgh-uk.md]: Yes...Fenwicks probably isn't there anymore!

Comment (EmilyUna65) [inline eats/edinburgh-uk.md]: yes @monicanpm there is alcohol at ALL of them :smile:

**Comment by carolynshin** [comment]
:sheep: it!

**Comment by monicanpm** [comment]
:sheep: it

**Pull request merged**